src/main/java/Daisy.java

```diff
@@ -0,0 +1,208 @@
+                        }
+                        createDeadline(separate_deadlines[0],separate_deadlines[1], true, false);
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println("Error! No event detected for deadline. Try again!");
+                    } catch (IllegalDeadlineFormatException e) {
+                        System.out.println("Error! Deadline entry is not following format. Try again!");
+                    }
+                    break;
+                case "event":
+                    try {
+                        String eventLine = separate_commands[1];
+                        int from = eventLine.indexOf(" /from ");
+                        int to = eventLine.indexOf(" /to ");
+                        if (from == -1 || to == -1 ) {
+                            throw new IllegalEventFormatException();
+                        }
+                        createEvent(eventLine.substring(0, from),eventLine.substring(from + " /from ".length(), to), eventLine.substring(to+" /to ".length()), true, false);
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println("Error! No event detected for event. Try again!");
+                    } catch (IllegalEventFormatException e){
+                        System.out.println("Error! Event entry is not following format. Try again!");
+                    }
+                    break;
+                default:
+                    try {
+                        throw new IllegalEntryException();
+                    } catch (IllegalEntryException e){
+                        System.out.println("Your input does not match any of my programs! Try again!");
+                    }
+                    break;
+            }
+            System.out.println(LINE_BREAK);
+            command = in.nextLine();
+        }
+
+        System.out.println(EXIT_PROMPT);
+        System.out.println(LINE_BREAK);
+
+        saveData();
+
+    }
+
+    public static void addItem(Task item) {
+        tasks.add(item);
+        System.out.println("Task received! The following has been added to your list of todos:\n" + item);
+        outputSize();
+    }
+
+    public static void deleteItem(int index) {
+        System.out.println("I see. The following task will be removed from your list:\n" + tasks.get(index));
+        tasks.remove(index);
+        outputSize();
+    }
+
+    public static void outputSize() {
+        System.out.println(String.format("Now you have %d tasks in your todo list.", tasks.size()));
+    }
+    public static void addItem(Task item, boolean printOut) {
+        tasks.add(item);
+
+        if (printOut) {
+            System.out.println("Task received! The following has been added to your list of todos:\n" + item);
+            outputSize();
+        }
+    }
+
+    public static void createTodo(String taskName, boolean printOut, boolean setDone) {
```

Comment:
Can consider creating a command abstract task and subclasses such as AddCommand for creating tasks